pom.xml

```diff
@@ -57,7 +57,7 @@
             <dependency>
                 <groupId>com.github.workerframework</groupId>
                 <artifactId>worker-framework</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
+                <version>4.0.0</version>
```

Comment:
This isn't right, updating this is part of https://portal.digitalsafe.net/browse/SCMOD-11991

Reply:
restored